modules/tensor_mechanics/test/tests/postprocessors/tests

```diff
@@ -0,0 +1,19 @@
+[Tests]
+  [./material_tensor_avg_test]
+    type = 'CSVDiff'
+    input = 'material_tensor_average_test.i'
+    csvdiff = 'material_tensor_average_test_out.csv'
+    requirement = 'The system shall be able to compute the volume average of RankTwoTensor component using MaterialTensorAverage'
```

Comment:
We try to keep implementation details out of requirements as much as possible. Our design should be test driven, e.g. in this hypothetical world we imagine the test (requirement) comes first and then we go write the implementation to fulfill that requirement. So in general I never have anything with capital letters in my requirements (except for the first world of course :smile:). So RankTwoTensor -> rank two tensor. Does that make sense? Please put a period at the end of the requirement.

Reply:
In this case this sentence is sufficient:

"The system shall be able to compute the volume average of a rank two tensor component."

You don't have to say how we're going to do it per se; you're just saying that the software system should be able to do it.

Reply:
@lindsay I'll keep this in mind. thanks for the suggestion!